extensions/tika/runtime/src/main/java/io/quarkus/tika/Content.java

```diff
@@ -0,0 +1,20 @@
+package io.quarkus.tika;
+
+public class Content {
```

Comment:
I would name this one `TikaResult`.

Reply:
I'm really not too keen to change `Content` to `TikaResult`, so now I'd have `parser.getResult(InputStream)` instead of `parser.getContent(InputStream)` the latter reflects better what the user is after IMHO... It looks natural to me to ask the parser to return a composite `Content` from a provided input stream